decidim-core/app/forms/decidim/base_form.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+module Decidim
+  # A base form object to hold common logic, like automatically adding as
+  # attributes the params sent as context by the `FormFactory` concern.
+  class BaseForm < Rectify::Form
```

Comment:
And @josepjaume, sometime they are needed as attributes, when there's an association between the object you're creating and the organization or the user.